op/op-create-groups/src/index.js

```diff
@@ -33,6 +33,7 @@ const operator = (configData, object) => {
 
   const ids = shuffle(globalStructure.studentIds);
   const struct = chunk(ids, configData.groupsize);
+
```

Comment:
It's better if you don't modify files when you don't need it. We could want to see the history of modification on `op/op-create-group` and it makes it easier to have the least number of commits possible.